commit f6645b55aa324d064f4469160e9a0e70af4bfa32 (refs/changes/59/1759/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-06T11:58:26-07:00 (2 years, 6 months ago)
    
    CNAPI-703 test-cnapi failing in nightly-1: DAPI-339 missed hard-filter-sick-servers reference in test config

diff --git a/config/test.json b/config/test.json
index 64b0fec..521082c 100644
--- a/config/test.json
+++ b/config/test.json
@@ -42,10 +42,12 @@
           "filter_large_servers": "",
           "filter_docker_min_platform": "",
           "filter_vm_limit": "",
+          "filter_owner_server": false,
           "disable_override_overprovisioning": "",
           "overprovision_ratio_cpu": "",
           "overprovision_ratio_ram": "",
           "overprovision_ratio_disk": "",
+          "minimum_free_disk": "",
           "weight_current_platform": "",
           "weight_next_reboot": "",
           "weight_num_owner_zones": "",
@@ -59,6 +61,7 @@
           "hard-filter-invalid-servers",
           "hard-filter-volumes-from",
           "hard-filter-reserved",
+          "hard-filter-min-free-disk",
           "hard-filter-vlans",
           "hard-filter-platform-versions",
           "hard-filter-traits",
@@ -69,10 +72,10 @@
           "calculate-ticketed-vms",
           "hard-filter-capness",
           "hard-filter-vm-count",
-          "hard-filter-sick-servers",
           "calculate-server-unreserved",
           "hard-filter-min-ram",
           "hard-filter-min-cpu",
+          "hard-filter-min-disk",
           "hard-filter-locality-hints",
           "hard-filter-owners-servers",
           ["or", "hard-filter-reservoir",
